src/libraries/System.IO.FileSystem/tests/ManualTests/ManualTests.cs

```diff
@@ -75,5 +75,65 @@ public static void Throw_FileStreamDispose_WhenRemoteMountRunsOutOfSpace()
                 destinationStream.Dispose();
             });
         }
+
+
+        const long InitialFileSize = 1024;
```

Comment:
I am not sure whether it is worth adding this tests with the new implementation. The failure path that was there before does not exist anymore.

Reply:
Wouldn't the test prevent that we don't go back to the previous implementation? However, since it needs to be run manually, I don't think it would catch a potential functional regression before it gets merged.

cc @carlossanlop

Reply:
I think it is very unlikely we would ever go back to the previous implementation. 

And if we ever change the implementation again, we should think about test coverage for any interesting failure modes that the change would introduce.

> since it needs to be run manually

Right, it makes the test very low value.